Makefile

```diff
@@ -458,10 +458,15 @@ TESTS += test_basic.static test_detail.static test_arm.static test_arm64.static
 TESTS += test_m68k.static test_mips.static test_ppc.static test_sparc.static
 TESTS += test_systemz.static test_x86.static test_xcore.static test_m680x.static
 TESTS += test_skipdata test_skipdata.static test_iter.static test_evm.static
-check: $(TESTS)
+check: $(TESTS) fuzztest
 test_%:
 	./tests/$@ > /dev/null && echo OK || echo FAILED
```

Comment:
probably should remove the `> /dev/null && echo OK || echo FAILED` and have just `./tests/$@`

Reply:
This is not my code.
I guess this is so that we can know which tests failed by looking at travis output